libfreerdp/codec/h264.c

```diff
@@ -71,7 +71,7 @@ struct _H264_CONTEXT_OPENH264
 {
 	ISVCDecoder* pDecoder;
 	ISVCEncoder* pEncoder;
-	SEncParamBase EncParamBase;
+	SEncParamExt EncParamExt;
```

Comment:
I had the impression that SEncParamExt isn't always available depending on the compilation of OpenH264, do I mixed up with something else ?